attributes/default.rb

```diff
@@ -8,13 +8,42 @@
 
 cookbook_name = 'barito-agent'
 
-default[cookbook_name]['produce_url'] = 'https://barito-router.golabs.io/produce'
-default[cookbook_name]['sources'] = []
-default[cookbook_name]['matches'] = []
+default[cookbook_name]['sources'] = [
+  {
+    type: 'systemd',
+    name: 'barito-journalctl',
+    path: '/run/log/journal',
+    raw_options: ['matches'],
+    matches: [{'_SYSTEMD_UNIT': 'syslog.service'}],
+    read_from_head: true,
+    storage: {
```

Comment:
I dont see variables storage and entry used anywhere

Reply:
This example assumes the users are using `fluentd-plugin-systemd` and I think I will move this to the readme referring your comment above.

Will make an update ASAP.